astlib/migrate_501_500.ml

```diff
@@ -787,13 +788,32 @@ and copy_module_expr_desc :
       Ast_500.Parsetree.Pmod_functor
         (copy_functor_parameter x0, copy_module_expr x1)
   | Ast_501.Parsetree.Pmod_apply (x0, x1) ->
-      Ast_500.Parsetree.Pmod_apply (copy_module_expr x0, copy_module_expr x1)
+      let x1 = copy_module_expr x1 in
+      let x1 =
+        match x1.pmod_desc with
+        | Pmod_structure [] ->
+            let pmod_attributes =
+              {
+                Ast_500.Parsetree.attr_name =
+                  {
+                    txt = "ppxlib.migration.keep_structure";
+                    loc = { x1.pmod_loc with loc_ghost = true };
+                  };
+                attr_payload = Ast_500.Parsetree.PStr [];
+                attr_loc = Location.none;
```

Comment:
Why `Location.none` instead of `{ x1.pmod_loc with loc_ghost = true }` like above?

Reply:
Because you are right! It should be `{ x1.pmod_loc with loc_ghost = true }` like above